versioned_docs/version-5.x/hiding-tabbar-in-screens.md

```diff
@@ -22,7 +22,7 @@ function App() {
     <Tab.Navigator>
       <Tab.Screen name="Home" component={HomeStack} />
       <Tab.Screen name="Feed" component={Feed} />
-      <Tab.Screen name="Settings" component={Settings} />
+      <Tab.Screen name="Notifications" component={Notifications} />
```

Comment:
```suggestion
      <Tab.Screen name="Settings" component={Settings} />
      <Tab.Screen name="Notifications" component={Notifications} />
```

Settings screen is still mentioned in the doc 😃

Reply:
Settings screen is still in HomeStack component, so I don't think it should be duplicated here.

Reply:
Ah yes, you are right!